sources/models/shared-defaults/aws-host-containers.toml

```diff
@@ -4,7 +4,7 @@ superpowered = true
 
 [metadata.settings.host-containers.admin.source]
 setting-generator = "schnauzer settings.host-containers.admin.source"
-template = "{{ ecr-prefix settings.aws.region }}/bottlerocket-admin:v0.9.0"
+template = "{{ ecr-prefix settings.aws.region }}/bottlerocket-admin:v0.9.2"
```

Comment:
Nit: Probably a note in the commit (and description) about why we updated from `0.9.0` to `0.9.2` could help :D!

Reply:
I think that would get rather verbose. In general we should just treat this as a typical dependency bump and I suspect we're going to update our host containers more regularly going forward.

Reply:
Added **Container changes** to the description though. 🚀